scheduler/test/cook/test/mesos/task.clj

```diff
@@ -324,6 +324,7 @@
                         :extract false
                         :value "file:///path/to/cook-executor"}}
         mesos-run-as-user nil]
+    (setup :config {:executor executor})
```

Comment:
If we `def-state` `executor-config`, then we can change this to a `with-redef` and can avoid calling `setup` of 'knowing' where in config to place the executor config.

Reply:
See comment above